src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -697,7 +697,7 @@ public static function debug_data() {
 		}
 
 		if ( function_exists( 'php_sapi_name' ) ) {
-			$php_sapi = php_sapi_name();
+			$php_sapi = PHP_SAPI;
```

Comment:
This change isn't the same as the rest in this PR or as the ticket describes.

Reply:
Thanks for this good catch, I was planning to make another ticket/PR with this change, but had this one spilled over. Reverted now.